frontend/src/stores/ftue-store.ts

```diff
@@ -68,10 +68,15 @@ export const useFTUEStore = defineStore('FTUE', () => {
     warningMessage.value = null;
   };
 
-  const nextStep = async () => {
+  const nextStep = async (call: Fetch) => {
     if (hasNextStep.value) {
       clearMessages();
       data.value.step = stepList[data.value.step].next;
+
+      call('metrics/ftue-step').post({
+        step_level: data.value.step,
+        step_name: stepList[data.value.step].title.replace('ftue.steps.', ''),
+      });
```

Comment:
What if for some reason that call fails? Do we need some error/exception handling here?

Reply:
If the api returns a bad response we will simply ignore the results. useFetch doesn't seem to list any exceptions used, and neither does the fetch api afaik.